locations/spiders/pizza_hut_il.py

```diff
@@ -0,0 +1,38 @@
+from scrapy.linkextractors import LinkExtractor
+from scrapy.spiders import CrawlSpider, Rule
+
+from locations.categories import Categories, apply_category
+from locations.hours import OpeningHours
+from locations.items import Feature
+
+DAYS_IL = {
+    "יום שני": "Mo",
+    "יום שלישי": "Tu",
+    "יום רביעי": "We",
+    "יום חמישי": "Th",
+    "יום שישי": "Fr",
+    "יום שבת": "Sa",
+    "יום ראשון": "Su",
+}
```

Comment:
Can you add this to the existing file with localized day names in it please?

Reply:
Done